.github/workflows/release.yml

```diff
@@ -49,7 +49,10 @@ jobs:
         working-directory: ./packages/vscode
         run: |
           npm i
-          npx ovsx publish -p ${{ secrets.OVSX_TOKEN }}
+          npm run build:ci:node
+          npx ovsx publish -p ${{ secrets.OVSX_TOKEN }} --target win32-x64 win32-ia32 win32-arm64 linux-x64 linux-arm64 linux-armhf darwin-x64 darwin-arm64 alpine-x64 alpine-arm64
```

Comment:
OVSX's documentation is a bit confusing on this and I'm not sure if it's right. I'd expect so since it's supposed to be equivalent to VSCE, though